deployment/node-problem-detector.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: apps/v1
 kind: DaemonSet
 metadata:
   name: node-problem-detector
+  labels:
+    app: node-problem-detector
```

Comment:
nit: s/app/k8s-app

Reply:
I found the label below is app: node-problem-detector. So I think app: node-problem-detector may be more suitable

Reply:
Ah, forgot about that. lgtm